string2pdb/README.md

```diff
@@ -12,10 +13,10 @@ Ensure you have the following installed:
 - Python 3 (on local server: use module load python/3.10/modulefile)
 
 # ARGUMENTS
-1. `<identifier>` (required, string): HUGO Gene Name of the target protein.
+1. `<identifier>` (required, string): Uniprot Accession Code of the target protein.
 
-2. `<threshold>` (optional, float): String_score threshold. Default for MAVISp: **0.7**. 
-   To adjust the threshold, specify with `-t <threshold>`.
+2. `<threshold>` (optional, float): String_score threshold. Default for MAVISp: **String_score > 0.7** AND **Database_score** > 0.

```

Comment:
is there any reason the database score is hardcoded in the script and not user-selectable?